src/styles/atoms/_atoms.link.scss

```diff
@@ -9,7 +9,7 @@
  * -------------------------------------------------------------------
  */
 
-a:not(.a-button) {
+a:not(.a-button, .a-button-outline, .a-button-negative, .a-button-transparent) {
```

Comment:
Toevoeging aan de fix van gisteren. Er bestaan natuurlijk nog andere `a-button`-klasses waarbij hetzelfde probleem voorkwam. Deze moeten natuurlijk ook toevoegd worden aan de `:not()`-selector.